samples/matter/thermostat/README.rst

```diff
@@ -103,33 +103,38 @@ Binding
 In this sample, the thermostat device prints simulated temperature data by default and it does not know the remote endpoints of the temperature sensors (on remote nodes).
 Using binding, the thermostat device updates its Binding cluster with all relevant information about the temperature sensor devices, such as their IPv6 address, node ID, and the IDs of the remote endpoints that contain the temperature measurement cluster.
 
-Matter thermostat build types
-=============================
+.. _matter_thermostat_custom_configs:
 
-The sample does not use a single :file:`prj.conf` file.
-Configuration files are provided for different build types, and they are located in the sample root directory.
-Before you start testing the application, you can select one of the build types supported by the application.
+Matter thermostat custom configurations
+=======================================
 
-See :ref:`app_build_additions_build_types` and :ref:`cmake_options` for more information.
+.. include:: /includes/sample_custom_config_intro.txt
 
-The sample supports the following build types:
+The sample supports the following configurations:
```

Comment:
Does it differ from the light bulb or it could be `.. include`-d here as well?

Reply:
+1

Reply:
Included.